src/python/grpcio_tests/tests/interop/server_ssl_cert_reload_test/helloworld_pb2_grpc.py

```diff
@@ -0,0 +1,47 @@
+import grpc
```

Comment:
Why include generated code for the test? Can the feature not be tested at a layer of abstraction below the generated code?

Reply:
done... tests no longer using these.